lib/danger/helpers/comments_helper.rb

```diff
@@ -151,17 +152,25 @@ def random_compliment
          "Yay.", "Jolly good show.", "Good on 'ya.", "Nice work."].sample
       end
 
-      def character_from_emoji(emoji)
+      def character_from_emoji(emoji, template: "github")
         emoji.delete! ":"
-        if emoji == "no_entry_sign"
-          "🚫"
-        elsif emoji == "warning"
-          "⚠️"
-        elsif emoji == "book"
-          "📖"
-        elsif emoji == "white_check_mark"
-          "✅"
-        end
+        
+        container = {
+          "github" => {
+            "no_entry_sign"    => "🚫",
+            "warning"          => "⚠️",
+            "book"             => "📖",
+            "white_check_mark" => "✅"
+          },
+          "bitbucket_server" => {
+            "no_entry_sign"    => "\u274C", # ❌
+            "warning"          => "\u2757", # ❗
+            "book"             => "\u2728", # ✨
+            "white_check_mark" => "\u2705"  # ✅
+          }
+        }
+
+        return container[template][emoji]
       end
```

Comment:
Good idea, but it needs some refactoring.
Would be nice to have a separate attribute like `@emoji_mapper` to map emoji character names into chars without re-defining the template. How about this?
```
class EmojiMapper
  DATA = {
    "github" => {
      "no_entry_sign"    => "🚫",
      "warning"          => "⚠️",
      "book"             => "📖",
      "white_check_mark" => "✅"
    },
    "bitbucket_server" => {
      "no_entry_sign"    => "\u274C", # ❌
      "warning"          => "\u26A0", # ⚠️
      "book"             => "\u2728", # ✨
      "white_check_mark" => "\u2705"  # ✅
    }
  }

  def initialize(template: "github")
    @template = (DATA.include? template) ? template : "github"
  end

  def map(emoji)
    emoji.delete! ":"
    DATA[template][emoji]
  end

  private

  attr_reader :template
end

EmojiMapper.new().map(":warning").to_s + " " + \
EmojiMapper.new(template: "bitbucket_server").map(":warning").to_s + " " + \
EmojiMapper.new(template: "bitbucket").map(":warning").to_s
```